testutil/jwt.go

```diff
@@ -0,0 +1,47 @@
+package testutil
+
+import (
+	"crypto/ecdsa"
+	"testing"
+	"time"
+
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jws"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+func TestCreateToken(tb testing.TB, id string, email string) jwt.Token {
```

Comment:
Sorry for the confusion. When helpers locate in the original package, we're trying to adopt the convention to prefix them with "test" (lowercase). 
But now we're already in the `testutil` package, we can save the prefix since it will self-explanatory, e.g. `testutil.CreateJWT`